docs/hub-tutorials/join-testnet.md

```diff
@@ -140,7 +140,7 @@ Cosmovisor requires the creation of the following directory structure:
 
 Install Cosmovisor and copy Gaia binary to genesis folder
 ```
-go install github.com/cosmos/cosmos-sdk/cosmovisor/cmd/cosmovisor@v1.3.0
+go install cosmossdk.io/tools/cosmovisor/cmd/cosmovisor@v1.5.0
```

Comment:
This downloaded:
go: downloading github.com/cosmos/cosmos-sdk v0.46.0-beta2.0.20230614103911-b3da8bb4e801

Any idea why a beta SDK 46 is being used?

Reply:
Lets keep 1.5.0 and I'll chase Marko / Julian in Slack. We can downgrade if need be later on.